algorithms.dtx

```diff
@@ -746,6 +746,7 @@ USA.
 % \newcommand{\algorithmicendrepeat}{\algorithmicend\ \algorithmicrepeat}
 % \newcommand{\algorithmicuntil}{\textbf{until}}
 % \newcommand{\algorithmicprint}{\textbf{print}}
+% \newcommand{\algorithmiccontinue}{\textbf{continue}}
```

Comment:
Thanks for your contribution, @filipesaraiva. Please, add a few lines describing the use of continue... Probably, the best place for this is in the for or in the while section of the document...

Reply:
Thanks @rbrito, I pushed more one commit adding documentation to `continue`. This pull request was updated.